Taarafo.Portal.Web/Taarafo.Portal.Web.csproj

```diff
@@ -1,11 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <Folder Include="Views\Bases\" />
```

Comment:
Why do we have this change?

Reply:
Removed because it is not necessary to include. VS only creates this include in .csproj only when you copy and paste a folder, not when you do create the folders from scratch. So it is obsolete code, and since its related to base a made sure its not included